### PR TITLE
chore: update metadata descriptions for static site resources

### PR DESCRIPTION
**Pull request by dannyrandall**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the Apache 2.0 License.

### CODE REVIEW

File: internal/pkg/deploy/cloudformation/stack/testdata/workloads/static-site.stack.yml [hunk 1]
```diff
@@ -133,7 +133,7 @@ Resources:
 
   TriggerStateMachineFunction:
     Metadata:
-      aws:copilot:description: A lambda that triggers the file-copying state machine
+      aws:copilot:description: A lambda that starts the process of moving files to the S3 bucket
```
Comment (huanjani) [inline internal/pkg/deploy/cloudformation/stack/testdata/workloads/static-site.stack.yml:136]: here and throughout, for consistency with the other templates?
```suggestion
      'aws:copilot:description': 'A lambda that starts the process of moving files to the S3 bucket'
```

File: internal/pkg/deploy/cloudformation/stack/testdata/workloads/static-site.stack.yml [hunk 2]
```diff
@@ -144,7 +144,7 @@ Resources:
 
   TriggerStateMachineFunctionRole:
     Metadata:
-      aws:copilot:description: An IAM Role for the lambda that triggers the file-copying state machine.
+      aws:copilot:description: An IAM Role for the lambda starts the process of moving files to the S3 bucket
```
Comment (huanjani) [inline internal/pkg/deploy/cloudformation/stack/testdata/workloads/static-site.stack.yml]:
```suggestion
      'aws:copilot:description': 'An IAM Role for the lambda that starts the process of moving files to the S3 bucket'
```

File: internal/pkg/deploy/cloudformation/stack/testdata/workloads/static-site.stack.yml [hunk 3]
```diff
@@ -167,7 +167,7 @@ Resources:
 
   TriggerStateMachineAction:
     Metadata:
-      aws:copilot:description: A custom resource to trigger the file-copying state machine
+      aws:copilot:description: A custom resource starts the process of moving files to the S3 bucket
```
Comment (huanjani) [inline internal/pkg/deploy/cloudformation/stack/testdata/workloads/static-site.stack.yml]:
```suggestion
      'aws:copilot:description': 'A custom resource that starts the process of moving files to the S3 bucket'
```

File: internal/pkg/template/templates/workloads/services/static-site/cf.yml [hunk 4]
```diff
@@ -169,7 +169,7 @@ Resources:
 
   TriggerStateMachineFunctionRole:
     Metadata:
-      aws:copilot:description: An IAM Role for the lambda that triggers the file-copying state machine.
+      aws:copilot:description: An IAM Role for the lambda starts the process of moving files to the S3 bucket
```
Comment (huanjani) [inline internal/pkg/template/templates/workloads/services/static-site/cf.yml]:
```suggestion
      'aws:copilot:description': 'An IAM Role for the lambda that starts the process of moving files to the S3 bucket'
```

File: internal/pkg/template/templates/workloads/services/static-site/cf.yml [hunk 5]
```diff
@@ -195,7 +195,7 @@ Resources:
 
   TriggerStateMachineAction:
     Metadata:
-      aws:copilot:description: A custom resource to trigger the file-copying state machine
+      aws:copilot:description: A custom resource starts the process of moving files to the S3 bucket
```
Comment (huanjani) [inline internal/pkg/template/templates/workloads/services/static-site/cf.yml]:
```suggestion
      'aws:copilot:description': 'A custom resource that starts the process of moving files to the S3 bucket'
```

**Review by huanjani — COMMENTED** [review]
just nits

**Review by Lou1415926 — APPROVED** [review]
LGTM! But let's merge this after the other fixes are merged :cool-crying

**Review by bvtujo — APPROVED** [review]

**Pull request merged**